src/api/contracts.ts

```diff
@@ -169,17 +169,19 @@ async function init(provider: Provider) {
     }, {}) as ContractsMapWProxy
 
     const { address: proxyAddress } = deployedContracts.DutchExchangeProxy
-    deployedContracts.DutchExchange = contractsMap.DutchExchange.at(proxyAddress)
+    deployedContracts.DutchExchange = await contractsMap.DutchExchange.at<Promise<DXAuction>>(proxyAddress)
```

Comment:
What if instead of specifying `template param` you change `SimpleContract` in `api/types.ts` interface to return `): Promise<T>`

Reply:
Also there's an instance of `HumanFriendly.at` in Tokens.ts

Reply:
:+1:

Reply:
after discussion w/me you anxo - this has been scrapped for synchronous `.at` calls